### DISCUSSION TITLE
join

### DISCUSSION THREAD

**Discussion by ohad1992** [dotnet/wpf#7952] [Q&A]
hi im new to WPF, i want to join you guys.
can someone explain me how i can stary?

**Comment by miloush** [comment]
Welcome! Great choice. There is some Getting Started documentation at https://learn.microsoft.com/en-us/dotnet/desktop/wpf/getting-started/. I can also recommend Adam Nathan's _WPF Unleashed_ and Charles Petzold's _Applications = Code + Markup_ books.

Let us know if you have any questions we could help with.

**Reply by ohad1992** [reply]
i already have an idea of WPF, i learned online.
how do i start to contributing? what is the project for?

**Reply by miloush** [reply]
This project repository contains the code of the WPF itself. You might want to read the [readme](https://github.com/dotnet/wpf/blob/main/README.md) and [Contributing Guide](https://github.com/dotnet/wpf/blob/main/Documentation/contributing.md). Note that there is more documentation in the [Documentation](https://github.com/dotnet/wpf/tree/main/Documentation) directory.

**Reply by ohad1992** [reply]
I read, it doesn't say much unfortunately

**Reply by miloush** [reply]
What information is it missing?

**Reply by ohad1992** [reply]
what is the project about?
there is a product or just people upload issues from own project?

**Reply by miloush** [reply]
The product is the WPF. We expect issues reporting bugs in WPF or suggestions how to improve it.
